### DISCUSSION TITLE
Chinese Translation Problem from a translator

### DISCUSSION THREAD

**Discussion by xubeiyan** [vuetifyjs/vuetify#17946] [Q&A]
As a translator，I use the Crowdin to translate the Vuetify Document from English to Chinese. But I founded all `Alert` like part will lose their content, even if I correctly translate it.

For example, in this url of [toolbar](https://vuetifyjs.com/eo-UY/components/toolbars/):

even if I had translated this 

> ::: warning
  When <code>v-btn</code>s with the <strong x-id="1">icon</strong> prop are used inside of <code>v-toolbar</code> and <code>v-app-bar</code> they will automatically have their size increased and negative margin applied to ensure proper spacing according to the Material Design Specification.

to 

> ::: warning
  当你在 <code>v-toolbar</code> 或者 <code>v-app-bar</code> 中使用具有 <strong x-id="1">icon</strong> 属性的 <code>v-btn</code> 组件时，Vuetify 会自动增加它们的大小和减少它们的外边距。以满足在Material Design 规范中对应的要求。

but in the production page, it will show a empty `Alert` like this:
![empty](https://github.com/vuetifyjs/vuetify/assets/4469398/07646e2f-71ff-43bf-b1e2-19ccfa30dab1)

I had translated about 14 component pages, I found many pages include this problem. Is there any solution?

**Comment by KaelWD** [comment]
#17516

I've just sent Crowdin another email about this, will update the issue when it's resolved.

**Comment by KaelWD** [comment]
This should be fixed now, you will probably have to update all the alert sections again
